enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
-
-And the aggregated root policy status would be: `Progressing` (this would include `ToApply`), `Succeeded`, or `Failed`.
-(The `TimeOut` status would not be reflected.)
-
-#### Handling managed cluster additions to the placement
-
-When a new managed cluster is added to the placement, if the rollout is `Progressing`, it will be handled with the
-current rollout where the `Placement` library will abstract this and return the cluster at the appropriate time
-according to the rollout strategy. If the rollout is `Succeeded`, the `governance-policy-propagator` will update the
-rollout status on the root policy back to `Progressing` and handle only the newly added managed clusters in a "miniature
-rollout" as described in the [Design section](#design).
-
-#### Handling complexities around the `PlacementBinding`
-
-There is some additional complexity brought into the rollout because policies are bound to `Placement` using
-`PlacementBindings`, so there could be multiple `Placements` bound to each policy. To remedy this, `All` and
-`Progressive` (which is per-cluster) is handled by pulling in all `PlacementDecisions` from all `Placements`. In the
-case of multiple `Placements` where `ProgressivePerGroup` is specified, this is handled by taking groups from one
-placement and then moving on to the next placement.
-
-Additionally, the `PlacementBinding` has specifications for selective policy enforcement to override the
-`remediationAction` and selectively apply policies to managed clusters. (See the
-[Selective Policy Enforcement enhancement](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-policy/28-selective-policy-enforcment).)
-The rollout will occur in tandem with selective policy enforcement. If selective policy enforcement is in play, the
-rollout will occur but would respect the remediation action overrides and only roll out enforcement to clusters selected
-by the `PlacementBinding`'s overrides.
-
-### User Stories
-
-#### Story 1
-
-As a system administrator, I want to know the status of a rollout for a policy and what clusters have been updated.
-
-- **Summary**
-
-  - Add a `RolloutStatus` to the `Policy` status in the CRD.
+sync controllers on managed clusters watch the associated managed cluster namespace on the hub for policy updates.
+
+### API Design
+
+#### Rollout Strategy API Integration
+
+Open Cluster Management policies will now have a new field of `spec.rolloutStrategy` that is based on the
+[rollout strategy API](https://open-cluster-management.io/concepts/placement/#rollout-strategy). An example is shown
+below.
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: sample-policy
+  namespace: policy-namespace
+spec:
+  disabled: false
+  remediationAction: enforce
+  rolloutStrategy:               # <-- New field
+    type: ProgressivePerGroup
+    progressivePerGroup:
+      minSuccessTime: 5m
+        progressDeadline: 10m
+        maxFailures: 2%
```

Comment:
Do we have some like fail-fast? wherever is fail, stop progressing

Reply:
> Do we have some like fail-fast? wherever is fail, stop progressing

The default is to stop rollouts on failures. These tuning points allow it to continue within customizable thresholds.